### PR TITLE
[RHACS] For parity with the openshift-docs distros

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Moved topic-map file in the topic-maps folder.

Cherry-pick in:
- `rhacs-docs-4.1`
- `rhacs-docs-4.2`
- `rhacs-docs-4.3`
- `rhacs-docs-4.4`

### CODE REVIEW

**Review by aireilly — COMMENTED** [review]
/lgtm

You will also need the linting updates here: https://github.com/openshift/openshift-docs/pull/53168/files

One of the new checks we run in Prow is a linter step.

File: _topic_maps/_topic_map.yml
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply _topic_maps/_topic_map.yml:1]: For RHACS, other branches have the same topic map so it is safe to cherrypick.

**Pull request merged**